packages/neo-one-node-vm/lib/Dispatcher.csproj

```diff
@@ -26,8 +26,8 @@
   </ItemGroup>
 
   <ItemGroup>
-    <ProjectReference Include="/Users/danielbyrne/neo/src/neo/neo.csproj" />
-    <ProjectReference Include="/Users/danielbyrne/neo-vm/src/neo-vm/neo-vm.csproj" />
+    <ProjectReference Include="/Users/spencercorwin/neo/src/neo/neo.csproj" />
+    <ProjectReference Include="/Users/spencercorwin/neo-vm/src/neo-vm/neo-vm.csproj" />
```

Comment:
sorry chief there can only be ONE of us who has to change this line after every rebase and I already called dibs.